ADDED issue template for bug report

Caketime opened the pull request:
ADDED basic code of conduct
<img width="956" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/85681889/e140756b-c990-4005-be53-21d3cfe287f7">
<img width="923" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/85681889/7b31f974-9908-451c-85b2-8ec2598496dd">
<img width="1036" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/85681889/a71f360c-9aba-4353-93f6-f2fc628763b4">

juliusmarminge commented on .github/ISSUE_TEMPLATE/BUG-REPORT.yml:
```diff
@@ -0,0 +1,47 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
```
unconfirmed_bug is probably more used and then we switch to bug once its reproduced

Thats how we do in t3-oss and trpc atleast

Caketime commented:
Force pushed an update so it stays a single commit.

Changes: Label "bug" -> "uncofirmed_bug"

markflorkowski reviewed (changes requested):
I'd also like to have "Reproduction Repository" and "Reproduction Steps" fields. They can be optional. 

Otherwise looks good!

Caketime commented:
- Added optional "Reproduction Repository" and "Reproduction Steps" fields

- Updated the "what did you expect to happen" description from "What did you expect to happen?" to "Please describe what you expected to happen."
